src/Raygun.NetCore.Tests.Blazor.WebAssembly/RaygunExceptionCatcherTests.cs

```diff
@@ -1,17 +1,16 @@
-﻿using CloudNimble.Breakdance.Blazor;
+﻿using Bunit;
+using CloudNimble.Breakdance.Blazor;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace Raygun4Net.Tests.Blazor.WebAssembly
+namespace Raygun.NetCore.Tests.Blazor.WebAssembly
```

Comment:
This was a recommendation by the IDE, to match the namespace definition with the folder name.